assets/js/blocks/cart-checkout/checkout/style.scss

```diff
@@ -106,11 +106,11 @@
 		padding-left: $gap-large;
 		padding-top: $gap;
 		padding-bottom: $gap;
-		line-height: $gap-large;
+		line-height: 1.375;
 
 		p,
 		.wc-block-product-metadata {
-			line-height: $gap;
+			line-height: 1.375;
```

Comment:
These line heights needed to be relative.

Reply:
It feels like these should have their own mixin helper.